src/common/FileSystem.cpp

```diff
@@ -1114,7 +1129,8 @@ static void InternalLoadPak(const PakInfo& pak, Util::optional<uint32_t> expecte
 				return;
 			}
 			realChecksum = crc32(*realChecksum, reinterpret_cast<const Bytef*>(&crc), sizeof(crc));
-			if (filename == PAK_DEPS_FILE) {
+			// can't reuse isLegacy in lambda
```

Comment:
You can if you capture it in the capture stanza like so:

		zipFile.ForEachFile([&pak, &realChecksum, &pathPrefix, &hasDeps, &depsOffset, &isLegacy](Str::StringRef filename, offset_t offset, uint32_t crc) {